packages/adminrouter/extra/src/includes/server/open/master.conf

```diff
@@ -60,7 +60,7 @@ location /exhibitor/ {
     include includes/proxy-headers.conf;
 
     proxy_pass http://exhibitor/;
-    proxy_redirect http://$http_host/ $scheme://$http_host/exhibitor/;
+    proxy_redirect http://$http_host/ /exhibitor/;
```

Comment:
Do we need http_host? Would the following work?

proxy_redirect / /exhibitor/;

I will have to play around with a real cluster to figure this out.

Reply:
The first parameter is a pattern that is removed from the upstream `Location` header and replaced with the second parameter. So we receive `Location: http://hostname/platypus` from Exhibitor and want to change it to `Location /exhibitor/platypus`.

We could use regular expressions to match
```
proxy_redirect ~^http://[^/]+/ /exhibitor/$1;
```
but that doesn't seem like an improvement.